XML/TestCases/FunctionalTests-Storage.xml

```diff
@@ -345,8 +345,9 @@
     <test>
         <testName>STORAGE-VHD-ADD-DIFF-DISK-IDE</testName>
         <setupScript>.\TestScripts\Windows\SETUP-DiffDiskGrowth.ps1</setupScript>
-        <testScript>.\TestScripts\Windows\STORAGE-DiffDiskGrowth.ps1</testScript>
+        <testScript>STORAGE-DiffDiskGrowth.ps1</testScript>
```

Comment:
Is it alright removing the path?

Reply:
Yes. It must be removed, otherwise, call RunTestScript() will be failed.
The below 3 lines comes from RunTestScript() in TestController.psm1.
$script = $CurrentTestData.TestScript
$scriptName = $Script.split(".")[0]
$scriptExtension = $Script.split(".")[1]

if the testScript is .\TestScripts\Windows\STORAGE-DiffDiskGrowth.ps1, the $scriptName and $scriptExtension are not right.

Reply:
OK, if this is right, then all other xml files should be updated. @LiliDeng  please create a new issue here. We should update all other xml files with omitting the path.